src/util/adb.js

```diff
@@ -111,8 +112,12 @@ export default class ADBUtils {
       .map((line) => line.replace('package:', '').trim())
       .filter((line) => {
         return (
-          line.startsWith('org.mozilla.fennec') ||
-            line.startsWith('org.mozilla.firefox')
+          firefoxApk ?
+            line.startsWith(firefoxApk) :
```

Comment:
Use `line === firefoxApk` instead of `line.startsWith(firefoxApk)`.
In practice there is no difference because the list is filtered again in the end, but let's be more explicit that this an exact match is intended:
https://github.com/mozilla/web-ext/blob/f9db35b30a7477cadd3f9101e1ee593efb2fe875/src/extension-runners/firefox-android.js#L340-L367